### ISSUE TITLE
Slide deck theme

### ISSUE THREAD

**Issue by bengalspice** [pgreischar/intro-data-capstone-biodiversity#2]
It would have been nice to see you pick a theme that relates to the data analysis rather than leaving the deck black and white, which makes the deck look stark and harsh on the eyes.

**Comment by pgreischar** [comment]
Agreed.   The only reason it was plain  b&w was to keep file size small and .pdf readability high.    I can quickly change this if needed.
